src/chrome/content/rules/Radioreference.com.xml

```diff
@@ -0,0 +1,23 @@
+<!--
+	Nonfunctional hosts in *.radioreference.com:
+
+	h: http redirect
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
```

Comment:
You can remove this comments.

Reply:
I've removed these in the latest commit.